packages/flutter/lib/src/services/text_input.dart

```diff
@@ -653,110 +643,6 @@ class RawFloatingCursorPoint {
   final FloatingCursorDragState state;
 }
 
-/// The current text, selection, and composing state for editing a run of text.
-@immutable
-class TextEditingValue {
```

Comment:
TextEditingValue was moved to its own file in order to not pollute this one, now that it's much larger.